src/main/java/com/microsoft/jenkins/azuread/integrations/casc/AzureAdMatrixAuthorizationStrategyConfigurator.java

```diff
@@ -0,0 +1,110 @@
+package com.microsoft.jenkins.azuread.integrations.casc;
+
+import com.microsoft.jenkins.azuread.AzureAdMatrixAuthorizationStrategy;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.security.AuthorizationStrategy;
+import hudson.security.Permission;
+import io.jenkins.plugins.casc.Attribute;
+import io.jenkins.plugins.casc.BaseConfigurator;
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.impl.attributes.MultivaluedAttribute;
+import io.jenkins.plugins.casc.model.CNode;
+import io.jenkins.plugins.casc.model.Mapping;
+import org.jenkinsci.plugins.matrixauth.AuthorizationContainer;
+import org.jenkinsci.plugins.matrixauth.integrations.PermissionFinder;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+@Extension(optional = true, ordinal = 2)
```

Comment:
No need for ordinal, its on the matrix-auth plugin because it was maintained separately in `configuration-as-code-support` plugin orginally
```suggestion
@Extension(optional = true)
```